utils/automation/fossologyscanner.py

```diff
@@ -140,38 +152,92 @@ def print_log_message(filename: str,
   report_file.close()
   return return_val
 
+def format_keyword_results_with_line_numbers(scanner:Scanners,format_results:FormatResult):
+  """
+  Format the keyword results with line numbers
+
+  :param: scanner : Scanner Scanner object
+  :param: format_results : FormatResult FormatResult object
+  :return: list of dicts with key as word and value as list of line numbers of the words
+  """
+  keyword_results = scanner.get_keyword_whole()
+  formatted_list_of_keyword_line_numbers = list()
+  for keyword_result in keyword_results:
+    list_of_scan_results = list(keyword_result.result)
+    words_to_search = [list_of_scan_results[i] for i in range(0,len(list_of_scan_results))]
```

Comment:
This line seems to be doing nothing. Can you please explain the need for this list comprehension?

Reply:
Yes it was redundant. I forgot to remove it. I have corrected it